dts/bindings/arm/nordic,nrf-uicr.yaml

```diff
@@ -7,3 +7,12 @@ include: base.yaml
 properties:
   reg:
     required: true
+
+  nfct-pins-as-gpios:
+    type: boolean
+    description: |
+      When enabled this property will configure pins dedicated to NFCT
+      peripheral as regular GPIOs.
```

Comment:
Hey is it worth adding a note mentioning that once this is set it can only be unset with an UICR wide erase? (that's the case for all uicr bits right?)

Reply:
added